population.h

```diff
@@ -72,8 +78,8 @@ double avg_fitness(const population& p);
 ///Calculates the fitness of inds in pop given a target env_value
 population calc_fitness(population p, const double &env_value, const double &sel_str);
 
-///changes the net of the nth individual to a given net
-void change_nth_ind_net(population& p, size_t ind_index, network n);
+///returns a pop with the net of the nth individual changed to a given net
+population change_nth_ind_net(population p, size_t ind_index, network n);
```

Comment:
To complement with my previous comment on this function, I would just make it a void return type as all the other `change` functions in population